devicetypes/smartthings/aeon-home-energy-meter.src/aeon-home-energy-meter.groovy

```diff
@@ -92,29 +133,58 @@ def zwaveEvent(physicalgraph.zwave.Command cmd) {
 }
 
 def refresh() {
+	log.debug "refresh()..."
 	delayBetween([
-		zwave.meterV2.meterGet(scale: 0).format(),
-		zwave.meterV2.meterGet(scale: 2).format()
+		encap(zwave.meterV2.meterGet(scale: 0)),
+		encap(zwave.meterV2.meterGet(scale: 2))
 	])
 }
 
 def reset() {
+	log.debug "reset()..."
 	// No V1 available
-	return [
-		zwave.meterV2.meterReset().format(),
-		zwave.meterV2.meterGet(scale: 0).format()
-	]
+	delayBetween([
+		encap(zwave.meterV2.meterReset()),
+		encap(zwave.meterV2.meterGet(scale: 0))
+	])
 }
 
 def configure() {
-	def cmd = delayBetween([
-		zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 4).format(),   // combined power in watts
-		zwave.configurationV1.configurationSet(parameterNumber: 111, size: 4, scaledConfigurationValue: 300).format(), // every 5 min
-		zwave.configurationV1.configurationSet(parameterNumber: 102, size: 4, scaledConfigurationValue: 8).format(),   // combined energy in kWh
-		zwave.configurationV1.configurationSet(parameterNumber: 112, size: 4, scaledConfigurationValue: 300).format(), // every 5 min
-		zwave.configurationV1.configurationSet(parameterNumber: 103, size: 4, scaledConfigurationValue: 0).format(),    // no third report
-		zwave.configurationV1.configurationSet(parameterNumber: 113, size: 4, scaledConfigurationValue: 300).format() // every 5 min
-	])
-	log.debug cmd
-	cmd
+	log.debug "configure()..."
+	if (zwaveInfo.model.equals("005F"))
+		delayBetween([
+			// Send combined power in watts to report group 1 every 5 seconds (if difference exceeds 10% or 50W)
+			encap(zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 2)),
+			encap(zwave.configurationV1.configurationSet(parameterNumber: 111, size: 4, scaledConfigurationValue: 5)),
```

Comment:
Report only happens if the change is more than 10%, right? Worried about some case where we're getting a report every 5s.

Reply:
I was seeing something similar with the energy value being reported too often, but not for the power. If we back off the reporting period for the power, we don't get any MeterReports until that period is met. For example, In our testing, when we turned off our test load, the app would not reflect the power change until the next reporting period (~5 minutes).
I'll look into this some more.